adminSiteClient/EditorCustomizeTab.tsx

```diff
@@ -376,6 +379,50 @@ class TimelineSection extends React.Component<{ editor: ChartEditor }> {
     }
 }
 
+@observer
+class MissingDataSection extends React.Component<{ editor: ChartEditor }> {
+    @computed get grapher() {
+        return this.props.editor.grapher
+    }
+
+    get missingDataStrategyOptions(): {
+        value: MissingDataStrategy
+        label: string
+    }[] {
+        const missingDataStrategyLabels = {
+            [MissingDataStrategy.auto]: "Automatic",
+            [MissingDataStrategy.hide]: "Hide charts with missing data",
+            [MissingDataStrategy.show]: "Show charts with missing data",
+        }
+
+        return Object.values(MissingDataStrategy).map((strategy) => {
+            return {
+                value: strategy,
+                label: missingDataStrategyLabels[strategy],
+            }
+        })
+    }
+
+    @action.bound onSelectMissingDataStrategy(value: string | undefined) {
+        this.grapher.missingDataStrategy = value as MissingDataStrategy
+    }
+
+    render() {
+        const { grapher } = this
+
+        return (
+            <Section name="Missing data">
+                <SelectField
+                    label="Missing data strategy"
```

Comment:
May clarify this a bit more:
```suggestion
                    label="Missing data strategy (for when some variables are missing data)"
```